plugins/destination/bigquery/client/spec.go

```diff
@@ -91,3 +117,15 @@ func isValidJson(content string) error {
 	}
 	return nil
 }
+
+func (Spec) JSONSchemaExtend(sc *jsonschema.Schema) {
+	sc.Properties.Value("batch_timeout").Default = "10s"
+}
+
+func (TimePartitioningOption) JSONSchemaExtend(sc *jsonschema.Schema) {
```

Comment:
this can be used instead:
```suggestion
func (TimePartitioningOption) JSONSchema() *jsonschema.Schema) {
```

Reply:
ah, right, you'll have to provide full spec manually in this case